tests/components/clarifai/test_api.py

```diff
@@ -0,0 +1,166 @@
+"""Tests for the Clarifai API."""
+from clarifai_grpc.grpc.api import resources_pb2, service_pb2
+from google.protobuf.json_format import Parse
+import pytest
+
+from homeassistant.components.clarifai.api import Clarifai
+from homeassistant.components.clarifai.const import CONCEPTS, DEFAULT, OUTPUTS
+from homeassistant.exceptions import HomeAssistantError
+
+from tests.async_mock import MagicMock, patch
+from tests.common import load_fixture
+
+DATA = "data"
+NAME = "name"
+VALUE = "value"
+
+
+class TestClarifaiAPI:
```

Comment:
We shouldn't test integration details directly. Set up the integration and assert core state, services and events.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations